tools/partyplanner

```diff
@@ -58,7 +58,7 @@ BIOS_MIB="4" # one per disk
 OVERHEAD_MIB="$((GPT_MIB * 2 + BIOS_MIB))"
 
 # The 'recommended' size for the EFI partition is 100MB but our EFI images are
-# under 1MB, so this will suffice for now. It would be possible to increase the
+# under 2MB, so this will suffice for now. It would be possible to increase the
```

Comment:
I have tried to build an image using a 1 MB EFI partition but that failed. Using a 2 MB partition worked fine.